src/user/views/adsManager/views/advanced/components/campaign/fields/FormatField.tsx

```diff
@@ -39,23 +39,22 @@ const FormatItemButton = (props: { format: CampaignFormat } & PriceProps) => {
   const { isEdit } = useIsEdit();
   const [, meta, format] = useField<CampaignFormat>("format");
   const [, , price] = useField<string>("price");
-  const [, bMeta, billing] = useField<Billing>("billingType");
+  const [, billingType, helper] = useField<Billing>("billingType");
 
   return (
     <ListItemButton
       disabled={isEdit}
       selected={meta.value === props.format}
       onClick={() => {
         format.setValue(props.format);
-        const found = props.prices.find((p) => {
-          return (
+        const found = props.prices.find(
+          (p) =>
             p.format === props.format &&
-            p.billingType === bMeta.value.toUpperCase()
-          );
-        });
+            p.billingType === billingType.value.toUpperCase(),
```

Comment:
Another `prices.find` with a `billingType.toUpperCase`... let's factor this logic out.